src/library/tests/test_views.py

```diff
@@ -293,3 +293,59 @@ def test_import_campaign_success(self):
 
         # all imported quests should be inactive for this campaign
         self.assertEqual(imported_library_campaign.quest_set.filter(visible_to_students=False).count(), 3)
+
+    def test_campaigns_library_list__filters_by_current_quests(self):
```

Comment:
We need a `test_campaigns_library_list__excludes inactive_campaigns()` because I believe that would have failed...